README.md

```diff
@@ -44,17 +43,27 @@ Documentation for the [latest published version](https://docs.rs/azure-functions
 
 # Getting Started
 
+## Install CMake
+
+Install [CMake](https://cmake.org/download/) and ensure the install location is added to the `PATH` environment variable.
```

Comment:
`cmake` can easily be installed on macOS and Linux using `brew`/the default package manager. Should we mention that here?

Reply:
That's a good idea.  I'll update.

Reply:
Pushed.